composer.lock

```diff
@@ -4,8 +4,60 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "d13e192b5c74a3c88dabcd20793e3ffc",
+    "content-hash": "ee42b9c93f311cc46106619f5c15f474",
```

Comment:
Should the `composer.lock` file be committed to this repository? My understanding of composer is that this lock file will make it problematic to install this package on Laravel 7 while being locked to Laravel 8?